src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -252,13 +253,17 @@ public void deployArmLessFar() {
     Pair<Double, Double> newPos =
         new Pair<Double, Double>(curPos.getFirst(), curPos.getSecond() - 0.5);
     liftPositionMap.replace(desiredPosition, newPos);
+
+    armController.setGoal(newPos.getSecond());
   }
 
   public void deployArmFurther() {
     Pair<Double, Double> curPos = liftPositionMap.get(desiredPosition);
     Pair<Double, Double> newPos =
         new Pair<Double, Double>(curPos.getFirst(), curPos.getSecond() + 0.5);
     liftPositionMap.replace(desiredPosition, newPos);
+
+    armController.setGoal(newPos.getSecond());
   }
```

Comment:
A thought I'm just now having about this - do we want to have it output the position it's changed to, via prints or something, so if for whatever reason something happens and Luka needs to change the position we know what he changed it to and can adjust the code using that as a starting point?